lib/attr_encrypted.rb

```diff
@@ -166,6 +175,30 @@ def attr_encrypted_options
     @attr_encrypted_options ||= superclass.attr_encrypted_options.dup
   end
 
+  def attr_encrypted_default_options
+    {
+      prefix:            'encrypted_',
+      suffix:            '',
+      if:                true,
+      unless:            false,
+      encode:            false,
+      encode_iv:         true,
+      encode_salt:       true,
+      default_encoding:  'm',
+      marshal:           false,
+      marshaler:         Marshal,
+      dump_method:       'dump',
+      load_method:       'load',
+      encryptor:         Encryptor,
+      encrypt_method:    'encrypt',
+      decrypt_method:    'decrypt',
+      mode:              :per_attribute_iv,
+      algorithm:         'aes-256-gcm',
```

Comment:
These look like good defaults to me. Do you think it'd be helpful if these defaults were displayed on the README all grouped together like this?